com.unity.cinemachine/Documentation~/CinemachineUsingSplinePaths.md

```diff
@@ -1,4 +1,6 @@
-# Using Spline paths
+# Constrain a camera to a path
+
+Use a Spline to constrain the movement of a camera along a predefined and customizable path.
```

Comment:
A hotlink here to Unity Splines would be most helpful

Reply:
Sure!

Reply:
hmmm... clicking on help for the Spline container gives me this https://docs.unity3d.com/2022.3/Documentation/Manual/script-SplineContainer.html so I'll skip it for now